routes/offlinetraces.js

```diff
@@ -30,7 +56,35 @@ module.exports = function (kafkaConfig) {
      */
     router.get('/:activityId', function (req, res) {
         var username = req.headers['x-gleaner-user'];
-        restUtils.processResponse(offlinetraces.findByActivityId(req.params.activityId, username), res);
+        restUtils.processResponse(offlinetraces.findByActivityId(req.params.activityId, username, false), res);
+    });
+
+
+    /**
+     * @api {post} /offlinetraces/:analisysId/kahoot Adds a new Kahoot OfflineTraces for a given versionId
+     * @apiName PostOflinetraces
+     * @apiGroup Oflinetraces
+     *
+     * @apiParam {String} analysisId The id of the analysis.
+     * @apiParamExample {} Request-Example:
+     *      An .xlsx file with the KAHOOT traces following the correct format:
+     *      https://kahoot.com/blog/2017/02/20/download-evaluate-kahoot-results-data/
+     *
+     * @apiSuccess(200) Success.
+     *
+     * @apiSuccessExample Success-Response:
```

Comment:
Same, the response doesn't have form of trace... where is the responses for example?

Reply:
True